bin/ovpn_getclient

```diff
@@ -5,25 +5,42 @@
 #
 
 if [ "$DEBUG" == "1" ]; then
-  set -x
+    set -x
 fi
 
 set -e
 
-source "$OPENVPN/ovpn_env.sh"
-cn=$1
+if [ -z "$OPENVPN" ]; then
```

Comment:
Why would `$OPENVPN` or `$EASRSA_PKI` ever be unset?  They are defined in the Dockerfile and overriden with `docker run -e ...`.  Is there a situation that makes this necessary?  Otherwise it's just adding noise to the script.